paper/basic_training.tex

```diff
@@ -646,131 +646,96 @@ \subsection{Thermostats}
-The instantaneous temperature is also known as the kinetic temperature and is typically computed via the equipartition theorem.
-It should be noted that the instantaneous temperature can be greater than, less than, or equal to the temperature corresponding to the average kinetic energy.
-The kinetic temperature should also be paid special attention to when describing the system.
-The kinetic temperature does \textbf{not} state ``the temperature is at some value $T$'', it merely states: ``the energy of the system is similar to that
-of a system at temperature $T$''.
-
-Several different thermostats are used to bring systems to a desired target temperature, and can be loosely grouped into three categories: deterministic, stochastic, and extended system (though some fall into multiple categories).
-Deterministic thermostats are usually known to scale the momenta or forces on the particles in the system based on how consistent these are with the target temperature.
-Stochastic thermostats behave similarly to the deterministic thermostats; however, they also involve a random force or momenta sampled out of a probability distribution that can that modifies the behavior of individual particles in a stochastic manner.
-Extended system thermostats are similar to what their name suggests --- the system has added variable(s) and an additional degree(s) of freedom relating to the thermostat.
-This emulates an external heat bath that can interact with the particles in the system, affecting their momenta in a predictable fashion.
+
+The temperature of a molecular dynamics simulation is typically measured using kinetic energies as defined using the equipartition theorem: $\frac{3}{2} N k_{\text{B}} T = \left<\sum_{i=1}^{N} \frac{1}{2} m_i v_i^2\right>$.
+The angled brackets indicate that the temperature is defined as a time-averaged quantity.
+If we use the equipartition theorem to calculate the temperature for a single snapshot in time of a molecular dynamics simulation instead of time-averaging, this quantity is referred to as the instantaneous temperature.
+The instantaneous temperature will not always be equal to the target temperature; in fact, in the canonical ensemble, the instantaneous temperature should undergo fluctuations around the target temperature.
+
+Thermostat algorithms work by altering the Newtonian equations of motion that are inherently microcanonical.
+Thus, it is preferable that a thermostat not be used if it is desired to sample dynamical properties such as diffusion coefficients; instead, the thermostat should be turned off after equilibrating the system to the desired temperature.
+However, some thermostats have been found to have little effect on dynamical properties, and they are commonly used during the production simulation as well\cite{Basconi:2013:J.Chem.TheoryComput.}.
```

Comment:
Yes, but these are still not the "true" dynamics of the system.  Maybe rather than say "little effect on the dynamical properties" say that the effect that the thermostat has on the properties is found to depend little on the thermostat parameters chosen and that this is deemed acceptable (but maybe more eloquently).  I think the same reference still applies.